<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GmshCfdMesh</class>
 <widget class="QWidget" name="GmshCfdMesh">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CFD Mesh by GMSH </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>14</number>
     </property>
     <property name="verticalSpacing">
      <number>8</number>
     </property>
     <property name="margin">
      <number>8</number>
     </property>
     <item row="2" column="1">
      <widget class="Gui::InputField" name="if_max">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolTip">
        <string extracomment="Select 0 to use default value"/>
       </property>
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="singleStep" stdset="0">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum" stdset="0">
        <double>1000000000.000000000000000</double>
       </property>
       <property name="unit" stdset="0">
        <string notr="true">mm</string>
       </property>
       <property name="decimals" stdset="0">
        <number>2</number>
       </property>
       <property name="value" stdset="0">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="Gui::InputField" name="if_min">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolTip">
        <string extracomment="Select 0 to use default value"/>
       </property>
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="singleStep" stdset="0">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum" stdset="0">
        <double>1000000000.000000000000000</double>
       </property>
       <property name="unit" stdset="0">
        <string notr="true">mm</string>
       </property>
       <property name="decimals" stdset="0">
        <number>2</number>
       </property>
       <property name="value" stdset="0">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cb_dimension">
       <property name="toolTip">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="l_dimension">
       <property name="text">
        <string>Element dimension:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="l_max">
       <property name="text">
        <string>Max element size:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>GMSH Parameters</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="l_min">
       <property name="text">
        <string>Min element size:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="margin">
      <number>8</number>
     </property>
     <property name="spacing">
      <number>8</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>GMSH</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gl_actions">
       <item row="0" column="0">
        <widget class="QTextEdit" name="te_output">
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="l_time">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Time:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::InputField</class>
   <extends>QLineEdit</extends>
   <header>Gui/InputField.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
